cmd/broker/fanout/main.go

```diff
@@ -51,8 +51,11 @@ type envConfig struct {
 	MaxStaleDuration time.Duration `envconfig:"MAX_STALE_DURATION" default:"1m"`
 
 	// MaxOutstandingBytes is the maximum size of unprocessed messages (unacknowledged but not yet expired).
-	// Default is 800Mb (~763Mi)
-	MaxOutstandingBytes int `envconfig:"MAX_OUTSTANDING_BYTES" default:"800000000"`
+	// Default is 400Mb
+	MaxOutstandingBytes int `envconfig:"MAX_OUTSTANDING_BYTES" default:"400000000"`
+
+	// MaxOutstandingMessages is the maximum number of unprocessed messages (unacknowledged but not yet expired).
+	MaxOutstandingMessages int `envconfig:"MAX_OUTSTANDING_MESSAGES" default:"4000"`
```

Comment:
Just in case, in current version, it seems _MaxOutstandingMessages_ parameter is [not as important](https://github.com/googleapis/google-cloud-go/blob/0f60593fc54894d430ff80fd087f91a65d46812e/pubsub/subscription.go#L505) in the async pulling case, although it's indeed good to pass expected value here should things change in future versions.